float64 swivel_cmd
float64 shoulder_cmd
float64 elbow_cmd
